src/test/java/teammates/test/cases/browsertests/InstructorSubmissionAdjustmentUiTest.java

```diff
@@ -38,7 +38,6 @@ protected void prepareTestData() {
 
         removeAndRestoreDataBundle(testData);
     }
-
```

Comment:
There originally was a blank line here (just that it didn't include trailing spaces).